src/sortStudents.ts

```diff
@@ -1,16 +1,57 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calculateAverage(grades: number[]): number {
+  return grades.reduce((sum, value) => sum + value) / grades.length;
```

Comment:
if you already use here `grades` to more meaningful would be rename `value` too.
```suggestion
  return grades.reduce((sum, grade) => sum + grade) / grades.length;
```

Reply:
Still actual)